src/reader.rs

```diff
@@ -155,10 +155,10 @@ impl<R: Read + Seek> VdbReader<R> {
 
     pub fn read_grid<ExpectedTy: Pod>(
         &mut self,
-        name: &'static str,
+        name: &str,
     ) -> Result<Grid<ExpectedTy>, ParseError> {
         let grid_descriptor = self.grid_descriptors.get(name).cloned();
-        let gd = grid_descriptor.ok_or(ParseError::InvalidGridName(name))?;
+        let gd = grid_descriptor.ok_or(ParseError::InvalidGridName(name.to_owned()))?;
```

Comment:
Doesn't clippy complain that this should be:

```suggestion
        let gd = grid_descriptor.ok_or_else(|| ParseError::InvalidGridName(name.to_owned()))?;
```

?

Reply:
No but let's do it anyway :)